crates/sui-storage/src/write_ahead_log.rs

```diff
@@ -85,7 +103,9 @@ pub trait WriteAheadLog<'a, C> {
     /// The caller is responsible for running the tx to completion.
     ///
     /// Recoverable TXes will remain in the on-disk log until they are explicitly committed.
-    async fn read_one_recoverable_tx(&'a self) -> SuiResult<Option<(C, Self::Guard)>>;
+    async fn read_one_recoverable_tx(
+        &'a self,
+    ) -> SuiResult<Option<(C, TransactionCommitPhase, Self::Guard)>>;
```

Comment:
We don't seem to be using the `TransactionCommitPhase` returned from using this function, so maybe we don't need to return it at all?

Reply:
Yes good catch, this was needed before a refactor I did. Not needed anymore